makefile_sample/Makefile

```diff
@@ -35,6 +35,14 @@ fmt-ci:
 	fgt gofmt -l $(SOURCES)
 .PHONY: fmt-ci
 
+imports:
+	goimports -w $(SOURCES)
+.PHONY: imports
+
+imports-ci:
+	fgt goimports -w $(SOURCES)
```

Comment:
i think in `-ci` mode should be `-l` instead of `-w`, otherwise it's never reported as an error. What do you think?